pyfar/dsp/dsp.py

```diff
@@ -1577,7 +1580,9 @@ def convolve(signal1, signal2, mode='full', method='overlap_add'):
     Returns
     -------
     Signal
-        The convolution result as a Signal object.
+        The convolution result as a Signal object. The
+        :py:mod:`cdim <pyfar._concepts.audio_classes>` matches the bigger cdim
+        of the two input signals.
```

Comment:
```suggestion
        The convolution result. The
        :py:mod:`cdim <pyfar._concepts.audio_classes>` matches the bigger cdim
        of the two input signals.
```
The type of the return data is already given in the line above.

Reply:
Yes, done